Tests.Integration.Presentation.Web/ItSystem/SystemRelationsTest.cs

```diff
@@ -366,6 +367,40 @@ public async Task Can_Get_SystemRelations_AssociatedWithContract()
             }
         }
 
+        [Fact]
+        public async Task Can_Get_SystemRelations_Defined_In_Organization()
+        {
+            //Arrange
+            var input1 = await PrepareFullRelationAsync(false, false, false);
+            var input2 = await PrepareFullRelationAsync(false, false, false);
+            var input3 = await PrepareFullRelationAsync(false, false, false);
+
+            //Make sure there are not relations before performing the paging test
+            DatabaseAccess.MutateEntitySet<SystemRelation>(repository =>
+            {
+                var systemRelations = repository.AsQueryable().ToList();
+                repository.RemoveRange(systemRelations);
+                repository.Save();
+            });
+
+            var relation1 = await SystemRelationHelper.PostRelationAsync(input1);
+            var relation2 = await SystemRelationHelper.PostRelationAsync(input2);
+            var relation3 = await SystemRelationHelper.PostRelationAsync(input3);
+
+            var expectedSequence = new[] { relation1.Id, relation2.Id, relation3.Id }.OrderBy(id => id).ToList();
+
+            //Act
+            const int pageSize = 2;
+            var firstPage = await SystemRelationHelper.GetRelationsDefinedInOrganization(TestEnvironment.DefaultOrganizationId, 0, pageSize);
+            var secondPage = await SystemRelationHelper.GetRelationsDefinedInOrganization(TestEnvironment.DefaultOrganizationId, 1, pageSize);
+
+            //Assert - first page with 2 items
+            Assert.Equal(expectedSequence.Take(2), firstPage.Select(x => x.Id));
+
+            //Assert - second page with 1 item since there are only 3 in total
+            Assert.Equal(expectedSequence.Skip(2).Take(1), secondPage.Select(x => x.Id));
```

Comment:
Ville det ikke være mere korrekt at bruge "pageSize" frem for "2"? Og så også i bruge "pageSize" under take? Den returnerer listen op til "pageSize" elementer. Så den vil give samme resultat og så fortæller testen også at du skipper en pageSize og tager så næste. Sammen med kommentaren der allerede er giver det mening

Reply:
jo til første comment med 1 er med vilje for jeg opstiller min "xpected"